app/src/main/java/com/spendster/presentation/homeScreen/Dashboard/ExpenseRecyclerViewAdapter.java

```diff
@@ -71,4 +78,12 @@ public int getItemViewType(int position) {
     private boolean isPositionHeader(int position) {
         return position == 0;
     }
+
+    public void bind(List<Expense> expenses) {
+        this.expenses = expenses;
+        if (expenses == null) {
+            Log.d(BuildConfig.MY_LOGS, "Expenses equal to null");
```

Comment:
@SimonMaha it's not so convenient to use the same log for everything. For example, if you would like to filter something for speciffic flow or screen...

Reply:
So it's better to use different tags for different features/ screens